client/reader/controller.js

```diff
@@ -59,8 +60,14 @@ function removeFullPostDialog() {
 	__fullPostInstance = null;
 }
 
-function ensureStoreLoading( store ) {
+function ensureStoreLoading( store, context ) {
```

Comment:
i didn't want to break apart the context, but yeah. could even just pass the start date directly, but then we have to have the code to pull things off the context everywhere.